### PR TITLE
Add SwiftSDK and RN tests to UnitTests scheme

### PR DESCRIPTION
**Pull request by brandonpage**

### CODE REVIEW

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_PR-Check">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by wmathurin** [comment]
Could you do the same for SalesforceReact?
For the tests to run however you will need to define an env variable `CI_USE_PACKAGER` with value `1` in the scheme like I did [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/Everything.xcscheme#L595).
And `npm start` needs to be run from the `libs/SalesforceReact` (for the RN packager).
If that's too much trouble, we could look at checking in the JS bundle instead.

**Comment by wmathurin** [comment]
Actually, you won't need to do a `npm start` since building React automatically causes the packager to  launch.

**Comment by brandonpage** [comment]
## Danger has errored
[!] Invalid `Dangerfile-Lib.rb` file: [31mWorkspace file not found at path `/Users/iosadmin/03b72f19/workspace/SalesforceMobileSDK.xcworkspace`[0m
```
 #  from .circleci/Dangerfile-Lib.rb:15
 #  -------------------------------------------
 #  if ENV.has_key?('JENKINS_URL')
 >    xcov.report(
 #        scheme: 'UnitTests',
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_CI-Result-SalesforceReact">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: .circleci/fastlane/Fastfile [hunk 1]
```diff
@@ -51,63 +51,61 @@ def testScheme(scheme)
...
+        clean: true
+    )
+  ensure
+    system('cp ../test_output/report.junit ../test_output/results.xml')
+
+    xcov(
+        workspace: "SalesforceMobileSDK.xcworkspace",
+        scheme: scheme,
+        output_directory: "xcov_output",
+        exclude_targets: "CocoaLumberjack.framework,SalesforceSDKCoreTestApp.app,SmartStoreTestApp.app,SmartSyncTestApp.app,SalesforceHybridSDKTestApp.app,SalesforceAnalyticsTestApp.app,RestAPIExplorer.app,AccountEditor.app,NoteSync.app,SmartSyncExplorerHybrid.app,SmartSyncExplorer.app,SmartSyncExplorerCommon.framework,RecentContactsTodayExtension.appex,Cordova.framework,SalesforceReact.framework,PromiseKit.framework,SalesforceSwiftSDKTestApp.app,SalesforceReactTestApp.app"
```
Comment (wmathurin) [inline .circleci/fastlane/Fastfile]: We shouldn't exclude `SalesforceReact.framework`, should we?

**Pull request closed without merging**